canonical-websites/cn.ubuntu.com
# Remove unused dependencies

We now use only a fraction of the Python dependencies, as we no longer use the admin area or a database.

QA
--

Simply run the site with `./run`, browse around, check it all works and looks the same as it did.
